README.md

```diff
@@ -149,6 +149,12 @@ with Creator("test.zim") as creator:
         creator.add_item(item2)
 ```
 
+#### Type hints
+
+`libzim` being a binary extension, there is no Python source to provide types information. We provide them as type stub files. When using `pyright`, you would normally receive a warning when importing from `libzim` as there could be discrepencies between actual sources and the (manually crafted) stub files.
+
+You can disable the warning via `reportMissingModuleSource = "none"`.
```

Comment:
Is it possible to disable the warning for all files but only a given module? Disabling this import by import is going to be painful, and disable it for all files and all modules is a bit risky.

Reply:
This is just a general suggestion so that users are aware of the warning and its pyright name. When I tested with scraperlib, I just added inline ignores (actually replaced the catchall that was there)